professor2-toolfile.spec

```diff
@@ -0,0 +1,29 @@
+### RPM external professor2-toolfile 1.0
+Requires: professor2
+%prep
+
+%build
+
+%install
+
+mkdir -p %i/etc/scram.d
+cat << \EOF_TOOLFILE >%i/etc/scram.d/professor2.xml
+<tool name="professor2" version="@TOOL_VERSION@">
+<client>
+<environment name="PROFESSOR2_BASE" default="@TOOL_ROOT@"/>
+</client>
+<use name="py2-numpy"/>
+<use name="py2-sympy"/>
```

Comment:
@davidlange6 , all of the py2-pippkgs tools are also separately available. So it is better to explicitly mention the dependency.
```
scram tool info py2-sympy
Tool info as configured in location /tmp/muzaffar/CMSSW_9_1_X_2017-03-07-2300
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++

Name : py2-sympy
Version : 1.0
++++++++++++++++++++
SCRAM_PROJECT=no
USE=py2-pippkgs
```